res/controllers/Reloop-Mixage.scripts.js

```diff
@@ -0,0 +1,812 @@
+                engine.softTakeoverIgnoreNextValue(effectUnit, "super1");
+            }
+        } else {
+            const effectSlot = `[EffectRack1_EffectUnit${unitNr}_Effect${focusedEffect}]`;
+            if (engine.getValue(effectSlot, "meta") !== Mixage.amountValue[group]) {
+                engine.softTakeoverIgnoreNextValue(effectSlot, "meta");
+            }
+        }
+    }
+};
+
+// The "disc" button that enables/disables scratching
+Mixage.scratchToggle = function(_channel, _control, value, _status, group) {
+    if (value === DOWN) {
+        Mixage.scratchPressed[group] = true;
+        Mixage.scratchTogglePressTimer = engine.beginTimer(400, function() {
+            Mixage.scratchTogglePressTimer = 0;
+        }, true);
+    } else {
+        Mixage.scratchPressed[group] = false;
+        if (Mixage.scratchTogglePressTimer !== 0) {
+            engine.stopTimer(Mixage.scratchTogglePressTimer);
+            Mixage.scratchTogglePressTimer = 0;
+            Mixage.stopLoopAdjust(group);
+            Mixage.scratchToggleState[group] = !Mixage.scratchToggleState[group];
+            Mixage.toggleLED(Mixage.scratchToggleState[group], group, "scratch_active");
+            if (Mixage.scrollToggleState[group]) {
+                Mixage.scrollToggleState[group] = !Mixage.scrollToggleState[group];
+                Mixage.toggleLED(Mixage.scrollToggleState[group], group, "scroll_active");
+            }
+        }
+    }
+};
+
+// The "loupe" button that enables/disables track scrolling
+Mixage.scrollToggle = function(_channel, _control, value, _status, group) {
+    if (value === DOWN) {
+        Mixage.stopLoopAdjust(group);
+        Mixage.scrollToggleState[group] = !Mixage.scrollToggleState[group];
+        Mixage.toggleLED(Mixage.scrollToggleState[group], group, "scroll_active");
+        if (Mixage.scratchToggleState[group]) {
+            Mixage.scratchToggleState[group] = !Mixage.scratchToggleState[group];
+            Mixage.toggleLED(Mixage.scratchToggleState[group], group, "scratch_active");
+        }
+    }
+};
+
+// The touch function on the wheels that enables/disables scratching
+Mixage.wheelTouch = function(_channel, _control, value, _status, group) {
+    const unitNr = script.deckFromGroup(group);
+
+    if (value === DOWN) {
+        Mixage.wheelTouched[group] = true;
+    } else {
+        Mixage.wheelTouched[group] = false;
+    }
+
+    if (Mixage.scratchByWheelTouch || Mixage.scratchToggleState[group]) {
+        if (value === DOWN) {
+            const alpha = 1.0 / 8.0;
+            const beta = alpha / 32.0;
+            engine.scratchEnable(unitNr, Mixage.scratchTicksPerRevolution, 33.33, alpha, beta);
+        } else {
+            engine.scratchDisable(unitNr);
+        }
+    }
+};
+
+// The wheel that controls the scratching / jogging
+Mixage.wheelTurn = function(_channel, _control, value, _status, group) {
+    const deckNr = script.deckFromGroup(group);
```

Comment:
can be moved to the else branch where it is needed

Reply:
good call I moved it to the else if branch